src/test/java/com/powsybl/openloadflow/dc/DcLoadFlowTest.java

```diff
@@ -563,6 +565,9 @@ void testDcApproxIgnoreG() {
     @Test
     void testDcResidualMismatchRemaining() {
         Network network = IeeeCdfNetworkFactory.create9();
+        // The purpose of this test is to verify the report when slack distribution mismatch is left on lack bus.
+        parametersExt.setSlackDistributionFailureBehavior(OpenLoadFlowParameters.SlackDistributionFailureBehavior.LEAVE_ON_SLACK_BUS);
+        parametersExt.setPlausibleActivePowerLimit(5000); // remove large groups from slack distribution to cause slack distribution failure
```

Comment:
groups -> generators

```suggestion
        parametersExt.setPlausibleActivePowerLimit(5000); // remove large generators from slack distribution to cause slack distribution failure
```